### DISCUSSION TITLE
[Feature] read-only dialog message for deleting photos when the drive is read-only

### DISCUSSION THREAD

**Discussion by Caspian-X** [immich-app/immich#13363] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Discord discussion starts [here](https://discord.com/channels/979116623879368755/994044917355663450/1294080721950081096)

I think it would be good to have a dialog/popup show up if the someone tries to delete an image if the drive is in read-only mode that tells them they cannot delete the photo for that reason. This is to keep the photos consistent between the drive (external library) and immich.

### Platform

- [X] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
We used to have this, but the special treatment for external libraries (and especially read-only ones) caused a lot of extra code and headaches, so we're trying to move away from that.

**Reply by Caspian-X** [reply]
Is there a way then to reset the database for deleted images so that images that were deleted via the immich frontend (but still exist in the read-only drive) show up in immich again?

**Reply by bo0tzz** [reply]
Running a library scan should do that.

**Reply by Caspian-X** [reply]
ok, that's working for me, thanks.
